File: exercicios/R/1101/1101.r
input = file('stdin','r')

while(TRUE){
    teste <- c()
    soma = as.integer(0)
    line = strsplit(readLines(input,n=1), " ")

    m = as.integer(line[[1]][1])
    n = as.integer(line[[1]][2])

    if(m > n){
        maior = as.integer(m)
        menor = as.integer(n)
    }else{
        maior = as.integer(n)
        menor = as.integer(m)
    }
    i = as.integer(menor)
    if(m == 0 || n== 0){
        break
    }
      
    while(i <= maior){
        soma = soma + i
        teste[i] <- i
        i = i + 1
    }
    if(soma > 0){
    teste <- na.omit(teste)
    write(cat(teste,sprintf("Sum=%d",soma)),'')}
}

File: exercicios/JavaScript/1101/1101.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){
    var teste = []
    var maior=0, menor=0, soma=0;
    var line1 = lines.shift().split(' ');
    var x = parseInt(line1[0]);
    var y = parseInt(line1[1]);

    if(x <= 0 || y <= 0){
        break;
    }

    if(x > y){
        maior = x;
        menor = y;
    }else{
        maior = y;
        menor = x;
    }

    for(let i=menor; i<=maior; i++){
        teste.push(i);
        soma += i;
    }

    console.log(teste.join(' ') + " Sum=" + soma)
}